beacon-chain/sharedstate/prometheus.go

```diff
@@ -0,0 +1,13 @@
+package sharedstate
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	StateCount = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "beacon_state_count",
+		Help: "Count the number of active beacon state objects.",
+	})
+)
```

Comment:
This had to be extracted, otherwise a metric with the same name was registered twice, causing a panic.